modules/odr_core/odr_core/crud/user.py

```diff
@@ -1,202 +1,12 @@
 # odr_core/crud/user.py
-from fastapi_sso import OpenID
 from sqlalchemy.orm import Session
 from sqlalchemy import update
 from sqlalchemy.exc import SQLAlchemyError
-from odr_core.models.user import User, UserSession
-from odr_core.models.team import Team
-from odr_core.schemas.user import UserCreate, UserUpdate
-from odr_core.config import settings
-from typing import Optional, List
-from datetime import datetime, timedelta, timezone
+from odr_core.models.user import User
+from typing import Optional
+from datetime import datetime, timezone
 from argon2 import PasswordHasher
 
-import uuid
-
-password_hasher = PasswordHasher()
```

Comment:
When I run `task data:migrate` I get an error in the create super user script from removing password_hasher here:

```
Repos/OMI-Data-Pipeline/modules/odr_datamodel/alembic/versions/7f105df0e1ed_create_superuser.py", line 14, in <module>
    from odr_core.crud.user import password_hasher
ImportError: cannot import name 'password_hasher' from 'odr_core.crud.user' (/home/zanya/Desktop/Development/Repos/OMI-Data-Pipeline/modules/odr_core/odr_core/crud/user.py). Did you mean: 'PasswordHasher'?
```
I think `alembic/versions/7f105df0e1ed_create_superuser.py` should be updated to remove the import, and so `password = None` for now rather than the hashed password since we don't support password logins anymore to fix this migration for now? Confirmed those changes locally allow `task data:migrate`  to succeed for me.